sample-runner/agate/package.json

```diff
@@ -30,15 +30,15 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^4.9.0",
-    "@enact/i18n": "^4.9.0",
+    "@enact/core": "^4.9.4",
+    "@enact/i18n": "^4.9.4",
     "@enact/agate": "enactjs/agate#develop",
```

Comment:
@0x64 , what do you think? Since enact version is 4.9.X, shouldn't we use agate 2.X, instead of develop branch?

Reply:
What about to use Enact 5.x version (so, develop branch) rather than use Agate 2.x? We won't make a dedicated branch for Agate 2.x and Agate's develop branch should match with Enact develop branch.